src/sources/http.rs

```diff
@@ -124,6 +134,27 @@ fn add_headers(
     events
 }
 
+fn add_query_params(
```

Comment:
This is duplicated exactly in `logplex`, maybe could be pulled into `util/http.rs`?

Reply:
:+1: I think that's a good idea. Maybe `src/sources/util/http.rs` since I think only sources would make use of it.